src/reach.lib.js

```diff
@@ -91,7 +91,7 @@ rch.detail.createForm = (name, action, target, values) => {
  *
  *       authorized: true if the transaction was authorized, else false.
  */
-rch.challenge = function(url, windowSize, iframeContainer, callback) {
+rch.challenge = function(url, windowSize, iframeContainer, callback, loading) {
```

Comment:
I think the naming in the README example is probably better:  `loaded`, since it is called when the iframe UI is loaded, not loading.  Also, can we make it clear that it's a callback function by calling it `loadedCallback`?